app/Http/Controllers/ConferencesController.php

```diff
@@ -111,14 +112,25 @@ public function show($id)
             return redirect('/');
         }
 
-        $talksAtConference = $conference->myTalks()->map(function ($talkRevision) {
-            return $talkRevision->talk->id;
+        $talks = auth()->user()->talks->map(function ($talk) use ($conference) {
```

Comment:
I feel like this should be in a presenter/transformer layer, but I couldn't find a similar convention for it. If there is one, please point me at it.

Reply:
Yah.. we could use API resources, or just a manual transformer for this. Since this doesn't end up in the API, I think I'd be more likely to make a basic transformer class for it.. something like

```php
class TalkTransformer
{
    public function __construct($talk)

    public function transform()
        return array
}
```

... probably